test/test.py

```diff
@@ -0,0 +1,53 @@
+# Execute this file from the test directory
+# python test.py
+
+#from os.path import dirname
+import sys
+import os.path
+import datetime
+import time
+
+sys.path.append(os.path.dirname(os.path.realpath(__file__)) + '/../client/')
+import boptest
+
+bop = boptest.Boptest()
+
+(siteref, modelname) = bop.submit('wrapped.fmu')
+
+time.sleep(3)
```

Comment:
instead of sleep, we should make the boptest client block and not return until the operation is fully complete.